config/ci_configs/slabplanet_aqua.yml

```diff
@@ -3,15 +3,13 @@ dt: "200secs"
 dt_cpl: "200secs"
 dt_save_to_sol: "10days"
 energy_check: true
-evolving_ocean: false
-h_elem: 6
+h_elem: 4
 bucket_albedo_type: "function"
 mode_name: "slabplanet_aqua"
 moist: "equil"
 precip_model: "0M"
 rad: "gray"
 start_date: "20100101"
 surface_setup: "PrescribedSurface"
-t_end: "20days"
-use_land_diagnostics: false
+t_end: "400secs"
```

Comment:
Is this one short enough to put it in normal CI?

Reply:
This one is in normal CI now too